packages/integrations/netlify/src/integration-functions.ts

```diff
@@ -85,6 +86,16 @@ function netlifyFunctions({
 				}
 			},
 			'astro:build:done': async ({ routes, dir }) => {
+				const functionsConfig = {
+						version: 1,
+						config: {
+							nodeModuleFormat: "esm"
+						}
```

Comment:
Is this documented somewhere? 👀

Reply:
No, sadly not! It's more of a stop-gap solution until we start outputting to ESM for all functions - but there's always sites that break in that migration, so we're pretty careful about it. There's something in the works on this though, i'm hoping we can remove this again in a couple months! Will let you know about it then :)